docs/contents/components/data-display/table/accessibility.ja.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Table
+description: "`Table`は、列のソートや行の選択やクリックイベント機能を備えたテーブルのコンポーネントです。"
+package_name: "@yamada-ui/table"
+order: 3
+with_description: true
+tab: アクセシビリティ
+---
+
+`Table`は、アクセシビリティに関して[WAI-ARIA - Table Pattern](https://www.w3.org/WAI/ARIA/apg/patterns/table/)に従います。
+
+`Table`に`aria-label`を設定すると、スクリーンリーダーによって読み上げられます。
+
+```tsx live=false
+<Table columns={columns} data={data} aria-label="DRAGON BALL characters" />
+```
+
+## キーボード操作
```

Comment:
The keyboard operation is fundamentally wrong.Please review everything.

Reply:
Should this be marked as 'Not applicable'?

>Keyboard Interaction
Not applicable.

https://www.w3.org/WAI/ARIA/apg/patterns/table/#keyboardinteraction

Reply:
Please refer here.

However, `Page Down`, `Page Up`, `Ctrl + Home` and `Ctrl + End` are optional and should be ignored.

https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/

Reply:
Fixed.
https://github.com/yamada-ui/yamada-ui/pull/3038/commits/ebde4228a99c32eed75da83773e89593ffa5e707